mysql/azuremysql/main.tf

```diff
@@ -0,0 +1,94 @@
+
+terraform {
+  required_version = "~>0.11"
+}
+
+provider "azurerm" {  
+  version = "~> 1.22"
+  # See documentation for more info: https://www.terraform.io/docs/providers/azurerm/auth/azure_cli.html
+  subscription_id = "00000000-0000-0000-0000-000000000000"
```

Comment:
IIUC the user of this Terraform setup would have to edit this line to the correct subscription ID. I believe if you remove this, then it will either prompt or use an environment variable.

Reply:
Correct, it will use the default subscription (if you have more than one). I'll remove it.